math/genvector/inc/Math/GenVector/Cartesian3D.h

```diff
@@ -190,10 +191,11 @@ public :
    template <class T2>
    explicit Cartesian3D( const Polar3D<T2> & v ) : fZ (v.Z())
    {
-      T rho = v.Rho(); // re-using this instead of calling v.X() and v.Y()
-      // is the speed improvement
-      fX = rho * std::cos(v.Phi());
-      fY = rho * std::sin(v.Phi());
+     using namespace std; 
+     const T rho = v.Rho(); // re-using this instead of calling v.X() and v.Y()
+     // is the speed improvement
```

Comment:
Could you not break the comments here? Maybe add it as a single comment a line before?